src/components/media/MediasSelectModal.jsx

```diff
@@ -291,16 +338,290 @@ const MediasSelectModal = ({
                 />
               </Center>
             )}
-            <LoadingButton
-              alignSelf="flex-end"
-              id="selectMedia"
-              isDisabled={!watch("selectedMedia")}
-              onClick={handleSubmit((data) => onProceed(data))}
-            >
-              Select
-            </LoadingButton>
-          </VStack>
-        </ModalFooter>
+          </div>
+        </Box>
+      </GridItem>
+    </Grid>
+  )
+
+  const FolderBreadcrumb = () => (
```

Comment:
could we shift this outside? this appears to be an independent component

Reply:
Makes sense, d71dd484!